providers/teams/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:10
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
```

Comment:
If you're binding the entire volume then there's no need for the `COPY` layer

Reply:
I'll keep until there's infra for dev mode with docker (as per our talk)